src/ontology/uberon-edit.obo

```diff
@@ -184,6 +184,7 @@ property_value: doap-SVNRepository "https://obo.svn.sourceforge.net/svnroot/obo/
 property_value: foaf-homepage "http://uberon.org" xsd:anyURI
 property_value: has_ontology_root_term UBERON:0000104
 property_value: has_ontology_root_term UBERON:0001062
+owl-axioms: Prefix(owl:=<http://www.w3.org/2002/07/owl#>)\nPrefix(rdf:=<http://www.w3.org/1999/02/22-rdf-syntax-ns#>)\nPrefix(xml:=<http://www.w3.org/XML/1998/namespace>)\nPrefix(xsd:=<http://www.w3.org/2001/XMLSchema#>)\nPrefix(rdfs:=<http://www.w3.org/2000/01/rdf-schema#>)\n\n\nOntology(\nDeclaration(Class(<http://purl.obolibrary.org/obo/UBERON_0001474>))\nDeclaration(Class(<http://purl.obolibrary.org/obo/UBERON_0007681>))\nDeclaration(Class(<http://purl.obolibrary.org/obo/UBERON_4200133>))\nDeclaration(Class(<http://purl.obolibrary.org/obo/UBERON_4200230>))\nDeclaration(Class(<http://purl.obolibrary.org/obo/UBERON_7500131>))\nDeclaration(Class(<http://purl.obolibrary.org/obo/UBERON_8500006>))\nDeclaration(ObjectProperty(<http://purl.obolibrary.org/obo/BFO_0000050>))\n\n############################\n#   Classes\n############################\n\n# Class: <http://purl.obolibrary.org/obo/UBERON_7500131> (<http://purl.obolibrary.org/obo/UBERON_7500131>)\n\nSubClassOf(<http://purl.obolibrary.org/obo/UBERON_7500131> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/UBERON_4200133> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000050> <http://purl.obolibrary.org/obo/UBERON_0007681>)))\n\n# Class: <http://purl.obolibrary.org/obo/UBERON_8500006> (<http://purl.obolibrary.org/obo/UBERON_8500006>)\n\nSubClassOf(<http://purl.obolibrary.org/obo/UBERON_8500006> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/UBERON_4200230> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000050> <http://purl.obolibrary.org/obo/UBERON_0001474>)))\n\n\n)
```

Comment:
You are trying to insert the following axiom:

> 'facial crest' **SubClassOf:** 'crest' **and** ('part of' **some** 'facial neural crest')

Such an axiom cannot be represented in “pure” OBO format, hence the introduction of this `owl-axioms` tag.

If you intended to say a 'facial crest' is both a 'crest' and something that is 'part of' some 'facial neural crest', you should do so in two independent **SubClassOf** axioms, there is no need for an ObjectIntersectionOf.

If you intended to say that any 'crest' that is 'part of' some 'facial neural crest' is a 'facial crest', then what you need is a **EquivalentTo** axiom, not a **SubClassOf** axiom. Such an axiom _is_ representable in “pure” OBO.

Reply:
@gouttegd do you have any guidance for when EquivalentTo should be used ('crest' and 'part of' some 'facial neural crest') or SubClaseOf ('crest', 'part of' some 'facial neural crest')? The facial crest is not a crest can pop up anywhere (as seems to be facial neural crest), it is specifically on the lateral side between the maxilla and zygomatic arch.

Reply:
There are some pages on the OBOOK about the issue (for example [here](https://oboacademy.github.io/obook/explanation/logical-axiomatization/#equivalent-class-logical-definitions) and [here](https://oboacademy.github.io/obook/explanation/subClassOf-vs-equivalentTo/)), though it’s nowhere near as complete as it should be IMHO.

But the most important rule of thumb, I believe, is: _When in doubt, use SubClassOf_. :) Whenever you are unsure which of SubClassOf or EquivalentTo is more appropriate, it can never be wrong to use SubClassOf (the worst that can happen is that you will make a statement that won’t be as “strong” as it should be, but that will be correct all the same).

Reply:
However, independently of the choice between SubClassOf and EquivalentTo, there may also be a problem with the term involved: are you sure this 'facial crest' is 'part of' some 'facial neural crest'?

'facial neural crest' is a part of the neural crest, an embryonic structure. That does not seem to fit well with the text definition.

Of note, the original term requested had proposed a logical definition involving 'facial skeleton', not 'facial neural crest'.